package.json

```diff
@@ -20,52 +20,42 @@
     "clean": "rm -rf ./src/XRP/Generated ./src/ILP/Generated ./dist ./build ./src/PayID/Generated",
     "pretest": "npm run clean && ./scripts/regenerate_protos.sh && ./scripts/regenerate_pay_id_api.sh && npm run lint && tsc --noEmit",
     "lint": "eslint . --ext .ts --fix",
+    "lintNoFix": "eslint . --ext .ts",
     "test": "nyc mocha",
     "prepublishOnly": "npm run test && npm run build"
   },
   "homepage": "https://github.com/xpring-eng/Xpring-JS#readme",
   "devDependencies": {
     "@openapitools/openapi-generator-cli": "^1.0.13-4.3.1",
     "@types/chai": "4.2.11",
-    "@types/chai-string": "1.4.2",
     "@types/google-protobuf": "3.7.2",
     "@types/mocha": "7.0.2",
     "@types/node": "^14.0.1",
-    "@typescript-eslint/eslint-plugin": "^2.18.0",
-    "@typescript-eslint/parser": "^2.18.0",
+    "@typescript-eslint/eslint-plugin": "^3.2.0",
+    "@typescript-eslint/parser": "^3.2.0",
     "chai": "4.2.0",
-    "chai-string": "1.5.0",
     "copyfiles": "^2.2.0",
-    "coveralls": "3.1.0",
-    "eslint": "^6.8.0",
-    "eslint-config-airbnb-base": "^14.0.0",
-    "eslint-config-airbnb-typescript": "^7.0.0",
-    "eslint-config-prettier": "^6.10.0",
-    "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-mocha": "^6.2.2",
-    "eslint-plugin-prettier": "^3.1.2",
+    "eslint": "^7.2.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.21.1",
+    "eslint-plugin-mocha": "^7.0.1",
+    "eslint-plugin-prettier": "^3.1.3",
     "grpc-tools": "1.9.0",
     "grpc_tools_node_protoc_ts": "^3.0.0",
     "mocha": "^7.0.1",
-    "mocha-lcov-reporter": "1.3.0",
     "nock": "^12.0.2",
     "nyc": "15.1.0",
     "prettier": "^2.0.4",
     "source-map-support": "0.5.19",
     "ts-node": "^8.6.2",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-config-standard": "^9.0.0",
-    "tslint-eslint-rules": "^5.4.0",
-    "typescript": "^3.9.2",
-    "typescript-eslint-parser": "22.0.0"
+    "typescript": "^3.9.3"
   },
   "dependencies": {
     "axios": "^0.19.2",
     "big-integer": "^1.6.48",
     "bignumber.js": "^9.0.0",
     "grpc": "^1.24.2",
     "grpc-web": "1.1.0",
-    "superagent": "^5.2.2",
```

Comment:
I just went ahead and removed all the dependencies that weren't actually being used.